src/lp_matrix_data.jl

```diff
@@ -50,9 +53,13 @@ The struct returned by [`lp_matrix_data`](@ref) has the fields:
    the value of `typemin(T)` is used.
  * `x_upper::Vector{T}`: the dense vector of variable upper bounds. If missing,
    the value of `typemax(T)` is used.
- * `c::Vector{T}`: the dense vector of linear objective coefficiennts
+ * `c::Vector{T}`: the dense vector of linear objective coefficients
  * `c_offset::T`: the constant term in the objective function.
  * `sense::MOI.OptimizationSense`: the objective sense of the model.
+ * `integers::Vector{Int}`: the list of column indices that are integer
```

Comment:
Is this list sorted ?

Reply:
Remember, this method is mainly pedagogical. Performance doesn't matter. Users just want vectors. Most probably don't know what `seachsorted` or `BitSet` are.